contrib/natss/pkg/controller/clusterchannelprovisioner/reconcile_test.go

```diff
@@ -254,7 +254,7 @@ func makeK8sService() *corev1.Service {
 			Selector: provisioners.DispatcherLabels(Name),
 			Ports: []corev1.ServicePort{
 				{
-					Name:       "http",
+					Protocol:   corev1.ProtocolTCP,
```

Comment:
Check this one: https://github.com/istio/istio/issues/13193.
Istio has a bug where if I name it then it wont work since it is an ExternalName service. So the change is to make it work in case pod that calls the channel has istio sidecar in it. Without istio sidecar it works irrespective of whether the port is named or not